src/git_changelog/cli.py

```diff
@@ -490,31 +582,39 @@ def main(args: list[str] | None = None) -> int:
     parser = get_parser()
     opts = parser.parse_args(args=args)
 
-    if opts.release_notes:
+    # Determine which arguments were explicitly set with the CLI
+    sentinel = _Sentinel()
```

Comment:
I believe we can simply do `sentinel = object()`, and get rid of the `_Sentinel` class, WDYT?

Reply:
Done, and it works.